sentry-core/src/clientoptions.rs

```diff
@@ -46,7 +47,11 @@ pub struct ClientOptions {
     pub in_app_include: Vec<&'static str>,
     /// Module prefixes that are never "in_app".
     pub in_app_exclude: Vec<&'static str>,
-    // TODO: Integration options
+    // Integration options
+    /// A list of integrations to enable.
+    pub integrations: Vec<Arc<dyn Integration>>,
```

Comment:
hm, thats why we have the additional boolean flag, also as discussed with daniel, the default integrations are added on top inside of `init`.

I do have one concern regarding the ordering. We should have some way to guarantee the order of integrations, so that the *attach a stacktrace to events* integration runs before the *trim and flag stackframes* integration.